app/views/layouts/application.html.erb

```diff
@@ -5,6 +5,7 @@
   <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
   <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
   <%= csrf_meta_tags %>
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css">
```

Comment:
You could add the `font-awesome` sass (or less) gem https://github.com/FortAwesome/font-awesome-sass instead of depending on a CDN. 
But it is not an important thing this!